unified-codes/apps/web/.storybook/config.js

```diff
@@ -1,5 +1,18 @@
-import { configure, addDecorator } from '@storybook/react';
+import { addDecorator, addParameters, configure } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 
+const webContext = require.context('..', true, /.stories\.(j|t)sx?$/);
+const libContext = require.context('../../../libs/ui/src/lib', true, /.stories\.(j|t)sx?$/);
+const parameters = {
+  options: {
+    storySort: {
+      method: 'alphabetical',
+      order: [],
+      locales: '',
+    },
+  },
+};
+
+configure([webContext, libContext], module);
```

Comment:
Hmm, I'm not sure if I like the idea of having the storybooks combined... was thinking it was nice to be able to view the web app components separately from the UI library (was thinking that for web, the UI library is essentially a package it doesn't know any implementation detail about)... does coupling them together add anything (can always run two storybooks, but does this remove the ability to view web components without the noise of the ui lib components)?

As the ui lib grows, it might contain components we want to keep around but which aren't used by web... just wondering if there might be a middle ground which allows us to run just the web storybook, just the ui storybook, and both storybooks (at the root, or something?). 

If not, I don't feel too strong on this either way, so I'm ok with it if everyone else thinks this is the best way to do it!